### PR TITLE
Fix 186

### LINKED ISSUE #186: reload when calling getpreviousview()
Hi, I would need this feature: I should be able to call a reload method when my interface is displayed as a consequence of the previousView method calls.. Do you think it is doable? It would be quite urgent..

### PR DESCRIPTION
**Pull request by samueleceroni**
fixes #186

### CODE REVIEW

File: src/main/java/nwoolcan/view/main/MainController.java [hunk 1]
```diff
@@ -42,7 +42,7 @@ public void menuViewDashboardClick(final ActionEvent event) {
     public void menuViewWarehouseClick(final ActionEvent event) {
         getController().setBreweryName("ciccio");
         getController().setOwnerName("ciccia");
```
Comment (Zeegomo) [inline src/main/java/nwoolcan/view/main/MainController.java]: why is this here?

Comment (samueleceroni) [reply src/main/java/nwoolcan/view/main/MainController.java]: it is in master, it does not concern my pr. don't block the flow for things like these.

Comment (samueleceroni) [reply src/main/java/nwoolcan/view/main/MainController.java]: @Zeegomo if you want to change that, open an issue and ask for explanations

Comment (Zeegomo) [reply src/main/java/nwoolcan/view/main/MainController.java]: anyway, it should not be there

Comment (Zeegomo) [reply src/main/java/nwoolcan/view/main/MainController.java]: > @Zeegomo if you want to change that, open an issue and ask for explanations

That's your part of view, it's pretty clear that this should not be on master
